bazel/repository_locations.bzl

```diff
@@ -615,6 +615,44 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         ],
         cpe = "cpe:2.3:a:webassembly_virtual_machine_project:webassembly_virtual_machine:*",
     ),
+    com_github_wasmtime = dict(
+        project_name = "wasmtime",
+        project_desc = "A standalone runtime for WebAssembly",
+        project_url = "https://github.com/bytecodealliance/wasmtime",
+        version = "0.21.0",
+        sha256 = "7874feb1026bbef06796bd5ab80e73f15b8e83752bde8dc93994f5bc039a4952",
+        strip_prefix = "wasmtime-{version}",
+        urls = ["https://github.com/bytecodealliance/wasmtime/archive/v{version}.tar.gz"],
+        release_date = "2020-11-05",
+        use_category = ["dataplane_ext"],
+        extensions = [
+            "envoy.access_loggers.wasm",
+            "envoy.bootstrap.wasm",
+            "envoy.filters.http.wasm",
+            "envoy.filters.network.wasm",
+            "envoy.stat_sinks.wasm",
+        ],
+        cpe = "cpe:2.3:a:webassembly_virtual_machine_project:webassembly_virtual_machine:*",
+    ),
+    com_github_wasm_c_api = dict(
+        project_name = "wasm-c-api",
+        project_desc = "WebAssembly C and C++ API",
+        project_url = "https://github.com/WebAssembly/wasm-c-api",
+        version = "d9a80099d496b5cdba6f3fe8fc77586e0e505ddc",
+        sha256 = "aea8cd095e9937f1e14f2c93e026317b197eb2345e7a817fe3932062eb7b792c",
+        strip_prefix = "wasm-c-api-{version}",
+        urls = ["https://github.com/WebAssembly/wasm-c-api/archive/{version}.tar.gz"],
+        release_date = "2019-11-14",
```

Comment:
This commit is a year old. There have been 10 commits since with the latest on Nov 3 2020. Any reason why we are not using the latest?

Reply:
This is included in Wasmtime as a submodule at the imported commit. There is no reason to diverge from that.

Reply:
Can't see where that specific commit is in https://github.com/bytecodealliance/wasmtime/blob/main/.gitmodules. If this old commit is the one we need to use please add a comment as to why this old dep needs to be used. In an upcoming version of https://github.com/envoyproxy/envoy/blob/master/tools/dependency/release_dates.py this will turn up as "aged".

Reply:
https://github.com/bytecodealliance/wasmtime/tree/v0.21.0/crates/c-api here you can see the submodule's commit. I will add the comment